_posts/en/newsletters/2020-05-13-newsletter.md

```diff
@@ -0,0 +1,163 @@
+  a0="FIXME"
+
+  q1="FIXME1"
+  a1="FIXME"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update to latest versions tuesday afternoon
+
+- [Eclair 0.4][] is a new major version release that upgrades Eclair's
+  major dependencies, adds support for the latest version of Bitcoin
+  Core, and deprecates Eclair Node GUI (instead users are encouraged to
+  run [Phoenix][] or [Eclair Mobile][]).
+
+- [C-Lightning 0.8.2.1][] is a new maintenance release that fixes an
+  incompatibility in large channels ("wumbo channels") between
+  C-Lightning and Eclair.  See the linked release notes for details.
+
+- [Bitcoin Core 0.20.0rc1][bitcoin core 0.20.0] is a release candidate
+  for the next major version of Bitcoin Core.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until version 0.21, about six months after the release of the upcoming
+version 0.20.*
+
+- [Bitcoin Core #16224][] displays all of the GUI's translated error
+  messages in both the locale's language and in English.  This can help
+  users find help and quickly describe the problem to developers.
+  Additionally, the English version of the error message is now what
+  gets written to the debug log, again to make it easier for developers
+  to comprehend the problem and provide assistance.
+
+- [C-Lightning #3659][] Watchtowers attempt 3: the full channelding FIXME:moneyball
+
+- [Rust-Lightning #539][] Require static_remotekey FIXME:dongcarl
+
+- [LND #4139][] extends the `gettransactions` and `listsweeps` RPCs to
+  allow passing start and end block heights to only retrieve
+  transactions confirmed within that block range.  A value of `-1` for
+  the end height may be used to also list unconfirmed transactions.
+
+## Footnotes
+
+[^increase-quote]:
+    "You keep the wallet connected to the computer, and the connected
+    computer runs a watch-only lightning wallet, and then communicates
+    with the hardware wallet for signatures on transactions that
+    strictly increase the balance of your channel." ---Excerpt from the
+    rough [transcript][snigirev ref] of a talk about hardware wallets by
+    Stephan Snigirev; transcript by Bryan Bishop
+
+[^outpoint-txid-spk]:
+    In the existing [BIP341][] specification of [taproot][topic
+    taproot], each input commits to the outpoints of every input
+    included in the transaction.  Outpoints are the concatenated txid
```

Comment:
maybe: transaction id (txid)

does "concatenated" here apply to both the txid and the vout?

Reply:
Removing the word "concatenated" to hopefully make it clearer.  "Txid" is one of our [unintroduced abbreviations](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#unintroduced-abreviations) in the style guide, and I continue to think most technical readers will be familiar with that term.

Reply:
> "Txid" is one of our [unintroduced abbreviations](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#unintroduced-abreviations) [](/bitcoinops/bitcoinops.github.io/blob/HEAD@{2020-05-11T11:38:24Z}/STYLE.md#unintroduced-abreviations) in the style guide, and I continue to think most technical readers will be familiar with that term.

Got it :+1: will keep that list in mind